sdk/testing/src/main/java/io/opentelemetry/sdk/testing/assertj/LogRecordDataAssert.java

```diff
@@ -167,6 +172,97 @@ public LogRecordDataAssert hasBody(@Nullable Value<?> body) {
     return this;
   }
 
+  public LogRecordDataAssert hasBodyField(String key, String value) {
+    return hasBodyField(key, Value.of(value));
+  }
+
+  public LogRecordDataAssert hasBodyField(String key, long value) {
+    return hasBodyField(key, Value.of(value));
+  }
+
+  public LogRecordDataAssert hasBodyField(String key, double value) {
+    return hasBodyField(key, Value.of(value));
+  }
+
+  public LogRecordDataAssert hasBodyField(String key, boolean value) {
+    return hasBodyField(key, Value.of(value));
+  }
+
+  public LogRecordDataAssert hasBodyField(String key, String... value) {
+    List<Value<?>> values = new ArrayList<>(value.length);
+    for (String val : value) {
+      values.add(Value.of(val));
+    }
+    return hasBodyField(key, Value.of(values));
+  }
+
+  public LogRecordDataAssert hasBodyField(String key, long... value) {
+    List<Value<?>> values = new ArrayList<>(value.length);
+    for (long val : value) {
+      values.add(Value.of(val));
+    }
+    return hasBodyField(key, Value.of(values));
+  }
+
+  public LogRecordDataAssert hasBodyField(String key, double... value) {
+    List<Value<?>> values = new ArrayList<>(value.length);
+    for (double val : value) {
+      values.add(Value.of(val));
+    }
+    return hasBodyField(key, Value.of(values));
+  }
+
+  public LogRecordDataAssert hasBodyField(String key, boolean... value) {
+    List<Value<?>> values = new ArrayList<>(value.length);
+    for (boolean val : value) {
+      values.add(Value.of(val));
+    }
+    return hasBodyField(key, Value.of(values));
+  }
+
+  // TODO: Make this public once AnyValue is no longer incubating
+  @SuppressWarnings({"unchecked"})
+  LogRecordDataAssert hasBodyField(String key, Value<?> value) {
```

Comment:
```suggestion
  public LogRecordDataAssert hasBodyField(String key, Value<?> value) {
```

Reply:
Had to keep the `@SuppressWarnings` due to the cast. Missing `public` seems like an oversight (or too much kotlin on the brain).